src/Assembler.hpp

```diff
@@ -285,6 +285,39 @@ class shasta::Assembler :
         // the number of virtual processors is used.
         size_t threadCount
     );
+    
+    // Loop over all alignments in the read graph
+    // to create vertices of the global marker graph.
+    // Throw away vertices with coverage (number of markers)
+    // less than minCoverage or more than maxCoverage.
+    // Also throw away "bad" vertices - that is, vertices
+    // with more than one marker on the same oriented read.
+    void createMarkerGraphVerticesGpu(
```

Comment:
I would prefer to put this declaration under `#ifdef SHASTA_BUILD_FOR_GPU`. I know it does not hurt, but it is just for readability, to make it clear that this function is only compiled in in that case.